consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/messagewrappers/PrepareTest.java

```diff
@@ -51,4 +52,23 @@ public void canRoundTripAPrepareMessage() {
     assertThat(decodedPrepare.getSignedPayload())
         .isEqualToComparingFieldByField(signedPreparePayload);
   }
+
+  @Test
+  public void testInterop() {
```

Comment:
Do we want the interop tests as part of the unit tests? Seems like something that perhaps should just be part of the reference RLP tests